Console.rb

```diff
@@ -0,0 +1,61 @@
+# Utility for writing to terminal, mostly colors and cursor movements.
```

Comment:
by convention, filenames should be lower case and snaked cased https://github.com/bbatsov/ruby-style-guide#snake-case-files